backend/fms_core/services/sample.py

```diff
@@ -1048,4 +1049,24 @@ def can_remove_sample(sample: Sample) -> Tuple[bool, List[str], List[str]]:
         was_processed = ProcessMeasurement.objects.filter(source_sample_id=sample.id).exists()
         is_removable = not is_child and not is_parent and not was_processed
 
-    return is_removable, errors, warnings
\ No newline at end of file
+    return is_removable, errors, warnings
+
+T = TypeVar("T")
+def defaultSelection_exceptedIDs(queryset: QuerySet[T], default_selection: bool, excepted_ids: list[int]) -> QuerySet[T]:
```

Comment:
This does not belong in sample service.

Reply:
We typically want in services stuff that can be both use from the API and from the other parts of the backend ... this is totally specific to a single endpoint.